app/controllers/FullResultsPageController.scala

```diff
@@ -112,8 +112,8 @@ class FullResultsPageController @Inject() (
                                 request.taxableProfit.toDouble,
                                 request.distributionsIncluded.flatMap(_.distributionsIncludedAmount).map(_.toDouble),
                                 request.associatedCompanies.associatedCompaniesCount,
-                                request.associatedCompanies.associatedCompaniesFY1Count,
-                                request.associatedCompanies.associatedCompaniesFY2Count
+                                None,
+                                None
```

Comment:
Is this correct? It seems that we don't account for TwoAsociatedCompanies anymore.

Reply:
Yeah, that was bug. Just fixed